KsApi/models/lenses/GraphCategoryLenses.swift

```diff
@@ -0,0 +1,67 @@
+import Prelude
+
+extension RootCategoriesEnvelope.Category {
+
+  public enum lens {
```

Comment:
Couldn't we do these lenses the way they were done in #243?

Reply:
Oh yes! Will do.